banking-app/scripts/start_docker.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+set -euo pipefail
+
+declare enclave_type=""
+
+function usage {
+    echo ""
+    echo "Start a CCF node docker in docker."
+    echo ""
+    echo "usage: ./start_docker.sh --serverIP <IPADDRESS> [--virtual] [--enclave]"
+    echo ""
+    echo "  --serverIP  string      The IP address of the primary CCF node"
+    echo "  --virtual   string      Run this in a virtual node"
+    echo "  --enclave   string      Run this in a SGX node"
+    echo ""
+    exit 0
+}
+
+function failed {
+    printf "Script failed: %s\n\n" "$1"
+    exit 1
+}
+
+# parse parameters
+
+if [[ $# -lt 3 || $# -gt 3 ]]; then
+    usage
+    exit 1
+fi
```

Comment:
Great spot - I have changed this to always check greater than the maximum number of parameters so that help works.  If you are happy, can you resolve.